Assets/MRTK.ThirdParty/MRTK-Quest/Scripts/Config/MRTKOculusConfig.cs

```diff
@@ -0,0 +1,297 @@
+        /// </summary>
+        public bool RenderAvatarHandsInsteadOfController => renderAvatarHandsInsteadOfControllers;
+
+
+#if OCULUSINTEGRATION_PRESENT
+        [Header("Prefab references")]
+        [SerializeField]
+        [Tooltip("Prefab reference for OVRCameraRig to load, if none are found in scene.")]
+        private OVRCameraRig ovrCameraRigPrefab = null;
+
+        /// <summary>
+        /// Prefab reference for OVRCameraRig to load, if none are found in scene.
+        /// </summary>
+        public OVRCameraRig OVRCameraRigPrefab => ovrCameraRigPrefab;
+#endif
+        [SerializeField]
+        [Tooltip("Use this if you want to manage the avatar hands prefab yourself.")]
+        private bool allowDevToManageAvatarPrefab = false;
+
+        /// <summary>
+        /// Use this if you want to manage the avatar hands prefab yourself.
+        /// </summary>
+        public bool AllowDevToManageAvatarPrefab => allowDevToManageAvatarPrefab;
+
+        [SerializeField]
+        [Tooltip("Prefab reference for LocalAvatar to load, if none are found in scene.")]
+        private GameObject localAvatarPrefab = null;
+
+        [Header("Pointer Configuration")]
+        [SerializeField]
+        [Tooltip("Controls which teleport mode is utilized by MRTK-Quest controllers." +
+                 "Note to use the official pointer, you must add a parabollic pointer to your pointer input profile that supports articulated hands.")]
+        private TeleportPointerMode teleportPointerMode = TeleportPointerMode.Custom;
+
+        /// <summary>
+        /// Controls which teleport mode is utilized by MRTK-Quest controllers.
+        /// Note to use the official pointer, you must add a parabollic pointer to your pointer input profile that supports articulated hands.
+        /// </summary>
+        public TeleportPointerMode ActiveTeleportPointerMode
+        {
+#if OVRPLUGIN_UNSUPPORTED_PLATFORM
+            // If the platform is not supported by oculus, we need to ensure we don't create a teleport pointer that can't be used.
+            get => TeleportPointerMode.None;
+#else
+            get => teleportPointerMode;
+#endif
+        }
+
+        [SerializeField]
+        [Tooltip("Custom teleport pointer prefab, to be managed directly by MRTK-Quest, given that MRTK doesn't currently officially support teleport with articulated hands.")]
+        private GameObject customTeleportPointerPrefab = null;
+
+        /// <summary>
+        /// Custom teleport pointer prefab, to be managed directly by MRTK-Quest, given that MRTK doesn't currently support teleport with articulated hands.
+        /// </summary>
+        public GameObject CustomTeleportPrefab => customTeleportPointerPrefab;
+
+        /// <summary>
+        /// Prefab reference for LocalAvatar to load, if none are found in scene.
+        /// </summary>
+        public GameObject LocalAvatarPrefab => localAvatarPrefab;
```

Comment:
Nit: Bring this up so that it's right under localAvatarPrefab  (i.e. pair the public/private closer together like is done in other locations)